no auth causes npe

**Describe the bug**
when authentication is not set, one will get npe out of 
        sdElement.addSDParam("subject", subject);

```
03.05.2024 14:06:09.529 [ERROR] [com.teragrep.lsh_01.RelpConversion] [http-input-handler-executor[T#1]] Unexpected error when sending a message: <Cannot invoke "String.length()" because "paramValue" is null> java.lang.NullPointerException: Cannot invoke "String.length()" because "paramValue" is null
        at com.teragrep.rlo_14.SyslogMessage.getEscapedParamValue(SyslogMessage.java:273)
        at com.teragrep.rlo_14.SyslogMessage.writeSDElements(SyslogMessage.java:265)
        at com.teragrep.rlo_14.SyslogMessage.toRfc5424SyslogMessage(SyslogMessage.java:248)
        at com.teragrep.rlo_14.SyslogMessage.toRfc5424SyslogMessage(SyslogMessage.java:207)
        at com.teragrep.lsh_01.RelpConversion.sendMessage(RelpConversion.java:191)
        at com.teragrep.lsh_01.RelpConversion.onNewMessage(RelpConversion.java:81)
        at com.teragrep.lsh_01.MessageProcessor.processMessage(MessageProcessor.java:139)
        at com.teragrep.lsh_01.MessageProcessor.run(MessageProcessor.java:121)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:840)
```

same goes for
```
        for (Map.Entry<String, String> header : headers.entrySet()) {
            headerSDElement.addSDParam(new SDParam(header.getKey(), header.getValue()));
        }
```

**Expected behavior**
no npe, empty element.

**How to reproduce**
send message without authentication set

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->

**Software version**
3.1.1

**Desktop (please complete the following information if relevant):**
 - OS: <!-- [e.g. iOS] -->
 - Browser: <!-- [e.g. chrome, safari] -->
 - Version: <!-- [e.g. 22] -->

**Additional context**
<!-- Add any other context about the problem here. -->

--- a/src/main/java/com/teragrep/lsh_01/RelpConversion.java
+++ b/src/main/java/com/teragrep/lsh_01/RelpConversion.java
@@ -121,7 +121,11 @@ public class RelpConversion implements IMessageHandler {
 
         SDElement headerSDElement = new SDElement("lsh_01_headers@48577");
         for (Map.Entry<String, String> header : headers.entrySet()) {
-            headerSDElement.addSDParam(new SDParam(header.getKey(), header.getValue()));
+            String headerValue = header.getValue();
+            if (headerValue == null) {
+                headerValue = "";
+            }
+            headerSDElement.addSDParam(new SDParam(header.getKey(), headerValue));
         }
         SDElement sdElement = new SDElement("lsh_01@48577");
         sdElement.addSDParam("subject", subject);

